libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -552,6 +552,11 @@ class YulUtilFunctions
 	/// signature: (data, len) -> data
 	std::string shortByteArrayEncodeUsedAreaSetLengthFunction();
 
+	/// @returns the name of a function that calculates slot and offset for index
+	/// Doesn't perform length checks, assumes that index is in bounds
+	/// signature: (array, index)
+	std::string longByteArrayStorageIndexAccessNoCheckFunction();
```

Comment:
I created this helper so I can use it in pop and as well in regular index access. Also can just have the same code twice if it is too much to create such a small function?